### ISSUE TITLE
Move remaining data needs to backend APIs

### ISSUE THREAD

**Issue by aristidesstaffieri** [stellar/freighter#1691]
We aim to move all data fetching to our backend in order to move away from Horizon and setup for a multi platform future.
We should find ways to moving the following remaining Horizon calls to the backend - 
`server.loadAccount(..pubkey..)` 
`server.accounts()`
`server.feeStats()`
`server.assets()`
`server.checkMemoRequired()`

Should we move these?
`Federation.Server.resolve()`

**Comment by JakeUrban** [comment]
Is this worth doing if we're going to move away from Horizon entirely? It may make sense to update this task to state that all calls for network data should be made on the backend (if thats the goal). Where we fetch that network data on the backend can be a task for stellar/freighter-backend.

**Comment by aristidesstaffieri** [comment]
Yeah I think this just needs re-wording. We should still make it so all data comes from an API call to the backend.
